pybamm/solvers/jax_solver.py

```diff
@@ -30,9 +30,9 @@ class JaxSolver(pybamm.BaseSolver):
 
     Parameters
     ----------
-    method: str
-        'RK45' (default) uses jax.experimental.odeint
-        'BDF' uses custom jax_bdf_integrate (see jax_bdf_integrate.py for details)
+    method: str, optional (see `jax.experimental.ode.odeint` for details)
+        * 'RK45' (default) uses jax.experimental.ode.odeint
+        * 'BDF' uses custom jax_bdf_integrate (see :func:`jax_bdf_integrate.py` for details)
```

Comment:
```suggestion
        * 'BDF' uses custom jax_bdf_integrate (see `jax_bdf_integrate.py` for details)
```

Reply:
Thanks I was just looking at that one too